demisto_sdk/commands/validate/tests/IF_validators_test.py

```diff
@@ -190,6 +204,109 @@ def test_IsCliNameReservedWordValidator_not_valid(reserved_word):
     )
 
 
+def test_IsFieldTypeChangedValidator_is_valid():
+    """
+    Given:
+        - IncidentFiled content items
+    When:
+        - run is_valid method
+    Then:
+        - Ensure that the ValidationResult returned
+          for the IncidentField whose 'type' field has changed
+        - Ensure that no ValidationResult returned when 'type' field has not changed
+    """
+    # not valid
+    content_item = create_incident_field_object(["type"], ["html"])
+    old_content_items = [create_incident_field_object(["type"], ["short text"])]
+    create_old_file_pointers([content_item], old_content_items)
+    assert IsFieldTypeChangedValidator().is_valid([content_item])
+
+    # valid
+    content_item.field_type = "short text"
+    assert not IsFieldTypeChangedValidator().is_valid([content_item])
+
+
+@pytest.mark.parametrize("unsearchable", (False, None))
+def test_UnsearchableKeyValidator_is_valid(unsearchable: bool):
+    """
+    Given:
+        - IncidentFiled content items
+    When:
+        - run is_valid method
+    Then:
+        - Ensure that the ValidationResult returned
+          for the IncidentField whose 'unsearchable' field is set to false or not or undefined
+        - Ensure that no ValidationResult returned when unsearchable set to true
+    """
+    # not valid
+    content_item = create_incident_field_object(
+        paths=["unsearchable"], values=[unsearchable]
+    )
+    assert UnsearchableKeyValidator().is_valid([content_item])
+
+    # valid
+    content_item.unsearchable = True
+    assert not UnsearchableKeyValidator().is_valid([content_item])
+
+
+def test_NameFieldPrefixValidator_is_valid_without_item_prefix():
+    """
+
+    Given:
+        - IncidentField content items
+    When:
+        - run is_valid method
+    Then:
+        - Ensure that the ValidationResult returned
+          for the IncidentField whose prefix name that not start with relevant pack name
+    """
+    # not valid
+    with ChangeCWD(REPO.path):
+        content_item = create_incident_field_object(pack_info={"name": "Foo"})
+        assert NameFieldPrefixValidator().is_valid([content_item])
+
+        # valid
+        content_item.name = "Foo CVE"
+        assert not NameFieldPrefixValidator().is_valid([content_item])
+
+
+@pytest.mark.parametrize(
+    "item_prefix, valid_prefix",
+    [
+        pytest.param(
+            ["Foo test", "Test Incident"], "Foo test CVE", id="itemPrefix is a list"
+        ),
+        pytest.param("Foo test", "Foo test CVE", id="itemPrefix is a str"),
+        pytest.param(None, "Foo CVE", id="no itemPrefix exists"),
+    ],
+)
+def test_NameFieldPrefixValidator_is_valid_with_item_prefix(
+    item_prefix: Optional[Union[List[str], str]], valid_prefix: str
+):
+    """
+    Given:
```

Comment:
please explain the differences between the cases.

Reply:
I'm using `pytest.param` with `id` parameter instead `docstring`